internal/cli/config/delete.go

```diff
@@ -80,22 +83,27 @@ func DeleteCmd(h *internal.Helper) *cobra.Command {
 				return errors.Trace(err)
 			}
 
-			err = t.Delete(profileName)
+			err = t.Delete(curProfileName)
```

Comment:
`curProfileName` is lowered, could it be deleted then?

Reply:
https://github.com/tidbcloud/tidbcloud-cli/pull/137/files#diff-a3a25ea6d49e8fbb116adc6f79d956d153bf828f2831cd60a148bced9195f009R143-R145

Reply:
viper will lowercase all config